spec/lib/email/sender_spec.rb

```diff
@@ -755,6 +755,7 @@
         end
 
         it 'should create a post reply key' do
+          SiteSetting.email_custom_headers = "Precedence: bulk"
```

Comment:
What's that for? 😃

Reply:
Woah that should not have been there haha, my bad.